### PR TITLE
Issue 852 - Add user register cypress tests

### PR DESCRIPTION
**Pull request by ckirby19**
### Issue link / number:
https://github.com/chaynHQ/bloom-frontend/issues/852

### What changes did you make?
Added the three test scenarios as needed

### Why did you make the changes?
To help catch any errors related to user sign in

### Did you run tests?
Yes, tests were run and passed

### CODE REVIEW

**Comment by ckirby19** [comment]
A few notes on the final text check that I used for each test:
- I would prefer to use a reference to the messages/auth/X.json files that have the actual text that should be output for these error messages, but could not work out how to do this. An attempt can be found in this [commit ](https://github.com/chaynHQ/bloom-frontend/pull/978/commits/a71e2726f157c28d629b438cacd039507677d2be)
- I expected that the invalidEmail error message would be used for the "test@test" example, but the actual error that is shown to the user is the generic 'There was an error setting up your account'

**Comment by ckirby19** [comment]
@eleanorreem PR for https://github.com/chaynHQ/bloom-frontend/issues/852

File: cypress/integration/create-user-existing.cy.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+describe('Create User', () => {
+  let username = Cypress.env('super_admin_email');
```
Comment (eleanorreem) [inline cypress/integration/create-user-existing.cy.tsx:2]: Looks great! Only thing is this should be 

`
let username = `cypresstestemail+${Date.now()}@chayn.co`;
`

**Review by eleanorreem — APPROVED** [review]
Apologies for the long delay ! This PR got lost in my notifications. Thanks so much for this ⭐ It looks great

**Pull request merged**